packages/opentelemetry-exporter-collector/README.md

```diff
@@ -19,18 +19,24 @@ npm install --save @opentelemetry/exporter-collector
 The CollectorTraceExporter in Web expects the endpoint to end in `/v1/trace`.
 
 ```js
-import { SimpleSpanProcessor } from '@opentelemetry/tracing';
+import { BatchSpanProcessor } from '@opentelemetry/tracing';
```

Comment:
Does the `concurrencyLimit` is meant only for BatchSpanProcessor ?. If not then I would not change the SpanProcessor, otherwise I would create new example

Reply:
It works for both.  If you use it with the simple processor you might set the limit a bit higher as it'll act as a sort of buffer size.

Reply:
As for the example, I changed it to use `BatchSpanProcessor` because I realized that if people actually use `SimpleSpanProcessor` with `CollectorTraceExporter` they will probably regret it.

That said, if you want me to revert the example or make a new one I can do that as well.  Just trying to help others avoid pain by giving a better example to start with.

Reply:
hmm maybe you right - it makes sense to have batch observer by default then